src/App.js

```diff
@@ -8,11 +8,40 @@ import './App.scss';
 // Notifications
 import { NotificationsPortal } from '@redhat-cloud-services/frontend-components-notifications';
 
+export const PermissionContext = createContext();
+
 class App extends Component {
 
+    constructor() {
+        super();
+        this.state = {
+            readPermission: undefined,
+            writePermission: undefined,
+            executePermission: undefined,
+            isOrgAdmin: undefined
+        };
+    }
+
+    handlePermissionUpdate = (hasRead, hasWrite, hasExecute) => this.setState({
+        readPermission: hasRead,
+        writePermission: hasWrite,
+        executePermission: hasExecute
+    });
+
     componentDidMount () {
         insights.chrome.init();
         insights.chrome.identifyApp('remediations');
+        window.insights.chrome.getUserPermissions().then(
+            allPermissions => {
+                const permissionList = allPermissions.map(permissions => permissions.permission);
+                this.handlePermissionUpdate(
+                    permissionList.includes('remediations:remediation:read'),
+                    permissionList.includes('remediations:remediation:write'),
+                    permissionList.includes('remediations:remediation:execute')
+                );
+            }
+        );
+        window.insights.chrome.auth.getUser().then((user) => user.identity.user.is_org_admin && this.setState({ isOrgAdmin: true }));
```

Comment:
This calback may complete at different time than the permission one. I.e. an org admin without permissions would see "you cannot use this" blip first and then it would work. Instead I think we should wait for both these calls to finish before dismissing the loading screen